.quickstart/quickstart.yml

```diff
@@ -3,12 +3,6 @@ schema_key: ga4_export_schema
 
 dbt_versions: ">=1.3.0 <2.0.0"
 
-destination_configurations:
-  databricks:
-    dispatch:
-      - macro_namespace: dbt_utils
-        search_order: [ 'spark_utils', 'dbt_utils' ]
-
```

Comment:
Since we don't install the spark utils package, there is no need for this config.